### PR TITLE
Fixing gitignore path, adding contributors.

### PR DESCRIPTION
**Pull request by subash-canapathy**
Hi vidhya please approve my changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # Java-OOP
 Java OOP Learning
+
+## Contributors
+- Vidhya
```
Comment (gvidhya) [inline README.md:5]: This is good

File: .gitignore [hunk 2]
```diff
@@ -20,4 +20,4 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
-/.idea/
+.idea/
```
Comment (gvidhya) [inline .gitignore:23]: Do we really need this change.

**Review by gvidhya — COMMENTED** [review]
Please respond.

File: .gitignore (hunk 2 above)
Comment (subash-canapathy) [reply .gitignore:23]: Its a //nit feel free to decline

**Pull request merged**